### PR TITLE
feat: add total_games and total_home_wins tests and methods

### PR DESCRIPTION
**Pull request by kenzjoy**
passing tests for methods total_games & total_home_wins

### CODE REVIEW

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -22,4 +22,23 @@ def highest_total_score
...
 
+  def total_games 
+    games = []
+    @game_teams.each do |row|
+      games << row[:game_id]  
+    end
+    games.uniq.count
+  end
+
+  def total_home_wins
```
Comment (ashuhleyt) [inline lib/stat_tracker.rb:33]: glad we were able to figure out this helper method together!

**Comment by naomiyocum** [comment]
nice helper methods!

**Review by ashuhleyt — APPROVED** [review]
looks good! left a few comments.

**Review by MoxieMax — COMMENTED** [review]
Comparing the home vs away goals is such a clever way to get those numbers

**Pull request merged**